### ISSUE TITLE
Seems like --yes stopped working

### ISSUE THREAD

**Issue by noherczeg** [lerna-lite/lerna-lite#485]
### Describe the bug

We are using the `--yes` option for publishing tasks, which worked in 1.13. but seems to be ignored in 1.15.x.

CI Job:

```
> new-publish:ci
> lerna publish --yes

lerna-lite notice cli v1.15.1
lerna-lite info ci enabled
lerna-lite info current project version 0.8.21
lerna-lite info Assuming all packages changed
? Select a new version (currently 0.8.21) (Use arrow keys)
❯ Patch (0.8.22) 
  Minor (0.9.0) 
  Major (1.0.0) 
  Prepatch (0.8.22-alpha.0) 
  Preminor (0.9.0-alpha.0) 
  Premajor (1.0.0-alpha.0) 
  Custom Prerelease 
  Custom Version 
Error: Process completed with exit code 130.
```

Corresponding npm script:

```
"new-publish:ci": "lerna publish --yes",
```

### Expectation

--yes should work

### Reproduction

Run `lerna publish --yes` in a CI environment, e.g.: Github Actions

### Lerna config and logs

<details><summary>lerna.json</summary><p>

```json
{
  "$schema": "node_modules/@lerna-lite/cli/schemas/lerna-schema.json",
  "packages": [
    "packages/@pandino/*"
  ],
  "npmClient": "npm",
  "version": "0.8.21",
  "ignoreChanges": [
    "**/__fixtures__/**",
    "**/__tests__/**",
    "**/helpers/**",
    "**/*.md"
  ]
}

```
</p></details>

<details><summary>lerna-debug.log</summary><p>

```txt
<!-- If you have a `lerna-debug.log` available, please paste it here -->
<!-- Otherwise, feel free to delete this <details> block -->
```
</p></details>

### Environment Info

lerna-lite version: 

```
"@lerna-lite/cli": "^1.15.1",
"@lerna-lite/run": "^1.15.1",
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/lerna-lite/lerna-lite/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://github.com/lerna-lite/lerna-lite#readme).
- [X] Check that there isn't [already an issue](https://github.com/lerna-lite/lerna-lite/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/lerna-lite/lerna-lite/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by noherczeg** [comment]
Found out that the `command.version` section in `lerna.json` is mandatory for `--yes` to work. This should either be fixed, or at least highlighted in the docs.

**Comment by ghiscoding** [comment]
if you know how to fix then please submit a PR, because the option you describe comes from the original code and I didn't change its behavior

**Comment by ghiscoding** [comment]
> Found out that the `command.version` section in `lerna.json` is mandatory for `--yes` to work. This should either be fixed, or at least highlighted in the docs.

There's maybe PR #431 that I replicated from Lerna recently, but you would have probably seen the error thrown and it's hard to confirm unless you look at your `lerna-debug.log`

**Comment by ghiscoding** [comment]
I tried it locally and you're most certainly facing the new error thrown in PR #431 with the following error

![image](https://user-images.githubusercontent.com/643976/220179980-d1bca9b5-7298-4b06-8be7-dc0f88930b73.png)

I updated the error message and also highlight the need for the `version` property, it should be more helpful with the following message

![image](https://user-images.githubusercontent.com/643976/220180284-e401010f-09bf-4b19-8479-81171fc56fd3.png)

Basically, the `version` was always a required config property but it wasn't as much highlighted before. Missing a `version` property could cause problems which is why it is now a required config property. I updated some of the README to better highlight this.
